client/src/components/Workflow/WorkflowList.vue

```diff
@@ -37,11 +37,10 @@
                     @onRestore="onRestore" />
             </template>
             <template v-slot:cell(tags)="row">
-                <Tags
-                    :index="row.index"
-                    :tags="row.item.tags"
+                <StatelessTags
+                    :value="row.item.tags"
                     :disabled="row.item.deleted"
-                    @input="onTags"
+                    @input="onTags($event, row.index)"
```

Comment:
```suggestion
                    @input="(tags) => onTags(tags, row.index)"
```

`$event` could hold any value. Providing a lambda is more explicit.